code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -529,6 +529,8 @@ datum
 				var/mob/living/M = target
 				if (!iscarbon(M) && !ismobcritter(M))
 					return
+				if (volume < 1)
```

Comment:
```suggestion
				if (volume_passed < 1)
```
Otherwise having 5u in a 400u container and giving people 5u sips would probably work

Reply:
Good catch, I misread the argument.